lib/crypto.js

```diff
@@ -234,5 +231,20 @@ Object.defineProperties(exports, {
     configurable: false,
     enumerable: true,
     value: constants
+  },
+
+  // Aliases for randomBytes are deprecated.
```

Comment:
Can you please add a comment: 

```js
// The ecosystem needs those to exists for backwards compatibility with
// ancient Node.js runtimes (0.10, 0.12).
```

Reply:
Added it. IMO this kind of makes it sound as if we couldn't remove these functions in the future though.

Reply:
Exactly, this is the sound of https://github.com/nodejs/node/pull/22519#issuecomment-415986849 comment.